stories/topics/proc-aws-generate-backup-data-file.adoc

```diff
@@ -2,17 +2,22 @@
 
 = Generating the backup data file
 
-The following commands create a directory, and populate it with an empty data template that, when completed, will be used during the backup. 
+The following commands create a directory, and populate it with an empty data template that, when completed, will be used during the backup.
 
 .Procedure
-. Create a folder to hold the configuration 
+. Create a folder to hold the configuration
 +
 [literal, options="nowrap" subs="+attributes"]
 ----
 $ mkdir command_generator_data
 ----
 . Populate the `command_generator_data` folder with the configuration file template.
-This creates the `backup.yml` file within the `command-generator_data` directory with `root:root_user` and `group` permissions.
+This creates the `backup.yml` file within the `command-generator_data` directory.
++
+[NOTE]
+====
+On Linux, any file or directory created by the command generator will be owned by `root:root` by default. To change the ownership of the files and directories, you can run the `sudo chmod` command after the files are created. Check the xref:con-tech-note-linux-files-owned-by-root[technical notes] for more information.
```

Comment:
generator is owned by...

For more information, read xref:con-tech-note-linux-files-owned-by-root[technical notes].